Задание 1-1 #1

ZemskovNikita opened the pull request:
1. Блок схема алгоритма.
2. Работа над ошибками.

ZemskovNikita committed:
Задание 1-1 #1

1.Разработана блок-схема алгоритма

ZemskovNikita committed:
Блок схема алгоритма. Работа над ошибками.

MarinaPaley reviewed (approved):
1. АлгОритм
2. Почему 2 файла в папке?????

ZemskovNikita committed:
Задание 1-1

MarinaPaley commented on Work_01/Task_1/Task 1-1/main.cpp:
```diff
@@ -0,0 +1,43 @@
+#define _USE_MATH_DEFINES 
+
```
Отступать нужно одну строку

MarinaPaley commented on Work_01/Task_1/Task 1-1/main.cpp:
```diff
@@ -0,0 +1,43 @@
+using namespace std;
+
+
+
```
Добавьте документацию

ZemskovNikita committed:
1-1

Создать консольное приложение, вычисляющее значения переменных по представленным в таблице формулам. Расчёт примера осуществить по заданным константам. Вывести на экран значения исходных данных, а также результат вычислений. Дополнить свой отчёт блок-схемой алгоритма.

ZemskovNikita committed:
#2-1

Создать консольное приложение для решения задачи, представленной в таблице, с использованием перечислимого типа. Выбор формулы вычисления зависит от пользователя. Данные для решения задачи так же вводит пользователь. Ввод необходимо проверять на правильность. Все результаты вывести на экран. Дополнить свой отчёт блок-схемой алгоритма.
Даны два числа. Вычислить их:
-сумму
-разность
-произведение
-частное

ZemskovNikita committed:
#3-1

Протабулировать заданную в таблице функцию. Использовать данные в таблице значения шага и интервала в качестве ввода пользователя для решения тестового примера. При невозможности расчёта функции в конкретной точке выводить её значение и надпись, означающую отсутствие решения.

При решении данного задания в MatLab необходимо построить график!

ZemskovNikita committed:
Add files via upload

ZemskovNikita committed:
Add files via upload

ZemskovNikita committed:
Add files via upload

ZemskovNikita committed:
Add files via upload
